src/pages/Login.jsx

```diff
@@ -33,33 +32,32 @@ const Login = () => {
             if (response.status === 200) {
                 const token = response.headers.get('jwt-token');
                 const tokenDecoded = jwtDecode(token);
-                const authorities = tokenDecoded.authorities
-                if(token != null){
-                    localStorage.setItem('token', token)
-                    localStorage.setItem('exp', tokenDecoded.exp)
-                    localStorage.setItem('tokenDecoded', tokenDecoded)
-                    if(authorities[1] === "ROLE_ROLE_STUDENT"){
-                        localStorage.setItem('role', authorities[1])
+                const authorities = tokenDecoded.authorities;
```

Comment:
sessionStorage.setItem('userId', response.data); is redundant. move this line to other location and you can use this once only

Reply:
I remove the redundancy of sessionStorage.setItem('userId', response.data) and moved it to where it will only be set once